BDKSwiftExampleWallet/View/SendView.swift

```diff
@@ -6,8 +6,111 @@
 //
 
 import SwiftUI
+import WalletUI
+import BitcoinDevKit
+
+class SendViewModel: ObservableObject {
+    
+    // Balance
+    @Published var balanceImmature: UInt64 = 0
+    @Published var balanceTrustedPending: UInt64 = 0
+    @Published var balanceUntrustedPending: UInt64 = 0
+    @Published var balanceConfirmed: UInt64 = 0
+    @Published var balanceTotal: UInt64 = 0
+    @Published var balanceSpendable: UInt64 = 0
+
+    // Sync
+    @Published var lastSyncTime: Date? = nil
+    @Published var walletSyncState: WalletSyncState = .notStarted
+    
+    // Transactions
+    @Published var transactionDetails: [TransactionDetails] = []
```

Comment:
Lets remove this because we won't have anything transaction related on this screen